### DISCUSSION TITLE
NER and OllamaSharp

### DISCUSSION THREAD

**Discussion by JackrabbitTek** [awaescher/OllamaSharp#137] [Q&A]
Has anyone tried to use OllamaSharp to interact with Named Entity Recognition?

**Comment by awaescher** [comment]
Not really sure what you mean with NER in this discussion. Parsing entities from text can be done with a good model, low temperature, json output (format=json) and a good system prompt. I did something like this a few times already but I am not sure if you'd define this as NER.
